modules/tensor_mechanics/test/tests/thermal_expansion/tests

```diff
@@ -35,12 +35,35 @@
   [./multiple_thermal_eigenstrains]
     type = 'Exodiff'
     input = 'multiple_thermal_eigenstrains.i'
-    exodiff = 'multiple_thermal_eigenstrains.e'
+    exodiff = 'multiple_thermal_eigenstrains_out.e'
     requirement = 'The tensor mechanics module shall have the capability to '
                   'combine multiple eigenstrains to correctly calculate an '
                   'eigenstrain tensor resulting from isotropic thermal '
                   'expansion.'
     design = 'ComputeThermalExpansionEigenstrain.md RankTwoAux.md'
     issues = '#7457'
   [../]
+
+  [./ad_constant_expansion_coeff]
+    type = 'Exodiff'
+    input = 'ad_constant_expansion_coeff.i'
+    exodiff = 'ad_constant_expansion_coeff_out.e'
+    requirement = 'The tensor mechanics module shall have the capability to '
+                  'calculate the eigenstrain tensor resulting from isotropic '
+                  'thermal expansion.'
+    design = 'ADComputeThermalExpansionEigenstrain.md'
+    issues = '#13091'
+  [../]
+  [./ad_constant_expansion_stress_free_temp]
+    type = 'Exodiff'
+    input = 'ad_constant_expansion_stress_free_temp.i'
+    exodiff = 'ad_constant_expansion_stress_free_temp_out.e'
+    requirement = 'The tensor mechanics module shall have the capability to '
+                  'calculate the eigenstrain tensor resulting from isotropic '
+                  'thermal expansion with an initial strain due to the '
+                  'difference between the stress free temperature and initial '
+                  'temperature of the material.'
+    design = 'ADComputeThermalExpansionEigenstrain.md'
+    issues = '#13091'
+  [../]
```

Comment:
`PetscJacobianTester`?

Reply:
Do you _still_ doubt the power of the mighty AD? ;-)

Reply:
Hm, glad I added those tests now. The existing tests are messed up (strain=small & ComputeFiniteStrainElasticStress in one and strain=finite & ComputeFiniteStrainElasticStress in the other)